LibraryComponents/Classes/Core/AndesTextFieldCode/View/AndesTextFieldCodeAbstractView.swift

```diff
@@ -38,6 +43,26 @@ class AndesTextFieldCodeAbstractView: UIControl, AndesTextFieldCodeView {
     internal func loadNib() {
         fatalError("This should be override by a subclass")
     }
+
+    override func didMoveToWindow() {
+        super.didMoveToWindow()
+        // This is necessary when setFocus() or removeFocus() are called, or variable 'text' is set, BEFORE the view is part of an active view hierarchy
+        if window != nil {
+            if let tempText = temporalText {
+                temporalText = nil
+                fillTextFieldsWith(text: tempText, from: 0)
+            }
+            if focusSet {
+                focusSet = false
+                let textField = getTextFieldsArray().first(where: { $0.text.isEmpty }) ?? getTextFieldsArray().last
+                textField?.becomeFirstResponder()
+            }
+            if focusRemoved {
+                focusRemoved = false
+                getTextFieldsArray().forEach { $0.resignFirstResponder() }
+            }
+        }
```

Comment:
No vale la pena tener todo wrappeado en ese `window != nil`
```suggestion
        super.didMoveToWindow()
        if window == nil {
            return
        }
        // This is necessary when setFocus() or removeFocus() are called, or variable 'text' is set, BEFORE the view is part of an active view hierarchy
        if let tempText = temporalText {
            fillTextFieldsWith(text: tempText, from: 0)
        }
        if focusSet {
            let textField = getTextFieldsArray().first(where: { $0.text.isEmpty }) ?? getTextFieldsArray().last
            textField?.becomeFirstResponder()
        }
        if focusRemoved {
            getTextFieldsArray().forEach { $0.resignFirstResponder() }
        }
        temporalText = nil
        focusSet = false
        focusRemoved = false
```